xsd-cli/src/main/java/io/ballerina/xsd/cli/XsdCmd.java

```diff
@@ -94,6 +94,10 @@ public void execute() {
             outStream.println(stringBuilder);
             return;
         }
+        if (input == null || input.isEmpty()) {
```

Comment:
Use `this` keyword when referencing class variables.
```suggestion
        if (this.input == null || this.input.isEmpty()) {
```